game/fx/evolution.go

```diff
@@ -72,7 +72,9 @@ func Evolution(card *match.Card, ctx *match.Context) {
 			1,
 			1,
 			false,
-			func(x *match.Card) bool { return x.SharesAFamily(card.Family) },
+			func(x *match.Card) bool {
+				return x.SharesAFamily(card.Family) || x.Name == "Innocent Hunter, Blade Of All"
```

Comment:
I think we should have a condition for cards that can be evolved from any family/race and check for that here instead of hardcoding the card name. Easier to add for future cards and less error-prone as you won't know that this function is dependent on the card's name if we in the future change how card names are formatted/stored

Reply:
could be something like `cnd.EvolveFromAnyFamily` or similar

Reply:
This should be fine. I don't remember any other card having such ability.

Reply:
Even so i'm still not a fan of matching on the name of a card like this. Not a great practice to follow in general. What if in the future we change the card names to be a localisation string to translate the card names into different languages, then this would break and you would have no idea as there's no types or references to the card involved, just a hardcoded string